server/graphql/v2/object/Application.js

```diff
@@ -91,6 +91,7 @@ export const Application = new GraphQLObjectType({
         if (userToken) {
           return {
             account: await req.loaders.Collective.byId.load(userToken.user.CollectiveId),
+            createdByAccount: () => req.loaders.Collective.byUserId.load(userToken.UserId),
```

Comment:
I don't understand the point, we already have `account` above, same thing.

Reply:
That's right, the confusion is on my side